src/Management/src/CloudFoundryCore/CloudFoundryServiceCollectionExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static class CloudFoundryServiceCollectionExtensions
         /// <param name="services">Service collection</param>
         /// <param name="config">Application Configuration</param>
```

Comment:
@ccheetham something like this?
```suggestion
        /// <param name="config">Application Configuration. Retrieved from the <see cref="IServiceCollection" /> if not provided</param>
```